src/main/java/uk/gov/hmcts/reform/lrdapi/domain/CourtType.java

```diff
@@ -37,6 +37,7 @@
     @Size(max = 16)
     private String courtTypeId;
 
+    @SuppressWarnings("java:S1700")
     @Column(name = "court_type", nullable = false)
     @Size(max = 128)
     private String courtType;
```

Comment:
instead of suppressing, why can't we rename the field to `typeOfCourt`?

Reply:
The only thing this suppresses is the [Sonar Issue](https://rules.sonarsource.com/java/RSPEC-1700) _"A field should not duplicate the name of its containing class"_ - however I am happy to make the changes, will require some refactoring elsewhere.

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-api/pull/285/commits/367a24a792c270447e7474def9789bc03e4d789e